src/System.Security.Cryptography.X509Certificates/src/Internal/Cryptography/Pal.Windows/X509Pal.PublicKey.cs

```diff
@@ -24,6 +24,9 @@ internal sealed partial class X509Pal : IX509Pal
     {
         const string BCRYPT_ECC_CURVE_NAME_PROPERTY = "ECCCurveName";
         const string BCRYPT_ECC_PARAMETERS_PROPERTY = "ECCParameters";
+        const int DSS_Q_LEN = 20;
+
+        private static readonly byte[] EmptyBuffer = new byte[DSS_Q_LEN];
```

Comment:
Nit: s_emptyBuffer

Reply:
Addressed in https://github.com/dotnet/corefx/pull/36838/commits/677b924f461fbcdb6366f45d7844cbc3cbafd6db